doc/chainloading.md

```diff
@@ -0,0 +1,23 @@
+# Chainloading
+
+Chainloading is a technique that allows one bootloader to call another bootloader as if the system had just booted up. [GNU GRUB](https://www.gnu.org/software/grub/) is one such bootloader which is commonly used for chainloading, as it presents a menu which you can use to select the OS you'd like to boot from. We're using `grub2` here.
+
+Create a file under `iso/boot/grub/grub.cfg` in the root directory of your OS's source tree. In it, put:
+
+```
+menuentry "BlogOS" {
```

Comment:
Perhaps we should use a generic name instead of "BlogOS" here (since this is independent of my blog series). Maybe something like `myOS`?